.github/workflows/ci-tests.yml

```diff
@@ -8,6 +8,20 @@ env:
   XCODE_VERSION: "15.0.0"
 
 jobs:
+  changes:
```

Comment:
Think we should rename this job to be specific in some way to pagination so that in the future we can have more jobs checking for changes for different packages like `apollo-ios` etc in the same way you are doing for pagination here.

Reply:
Perhaps more specific to running unit tests? 
You can setup multiple outputs for this job.

Right now, it's:

```yml
  changes:
    runs-on: ubuntu-latest
    outputs:
      pagination: ${{ steps.filter.outputs.project }}
    steps:
      - uses: actions/checkout@v4
      - uses: dorny/paths-filter@v2
        id: filter
        with:
          filters: |
            pagination:
              - 'apollo-ios-pagination/**'
              - 'Tests/ApolloInternalTestHelpers/**'
              - 'apollo-ios/**'
```

But it could be:

```yml
  changes:
    runs-on: ubuntu-latest
    outputs:
      pagination: ${{ steps.filter.outputs.project }}
      codegen: ${{ steps.filter.outputs.codegen }}
    steps:
      - uses: actions/checkout@v4
      - uses: dorny/paths-filter@v2
        id: filter
        with:
          filters: |
            pagination:
              - 'apollo-ios-pagination/**'
              - 'Tests/ApolloInternalTestHelpers/**'
              - 'apollo-ios/**'
            codegen:
              - 'apollo-ios-codegen/**'
              - 'Tests/ApolloInternalTestHelpers/**'
```

Reply:
Ah didn't realize that action would allow multiple outputs, then yea I think what you have here is great and we can add too it as needed. Thanks!

Reply:
Looks like it's working as it's intended, now that I'm mapping the right variable names 😅